src/libsync/syncfileitem.cpp

```diff
@@ -72,6 +72,7 @@ SyncFileItemPtr SyncFileItem::fromSyncJournalFileRecord(const SyncJournalFileRec
     item->_remotePerm = rec._remotePerm;
     item->_serverHasIgnoredFiles = rec._serverHasIgnoredFiles;
     item->_checksumHeader = rec._checksumHeader;
+    item->_remoteName = QString::fromUtf8(rec._serverPath);
```

Comment:
Shouldn't we have the same name `remoteName` for DB as well? Just to be consistent ...